packages/core/src/external/aws/lambda-logic/bundle-to-html.ts

```diff
@@ -1095,6 +1095,13 @@ function createConditionSection(conditions: Condition[], encounter: Encounter[])
 
     return aDate === bDate && aText === bText;
   })
+    .filter(condition => {
+      const snomedCode = condition.code?.coding?.find(coding =>
+        coding.system?.toLowerCase().includes(SNOMED_CODE)
+      )?.code;
+      const blacklistCodes = ["55607006"];
```

Comment:
please put this in a const so we know what it means - let's never use hardcoded magic string numbers in the future